<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_total">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_buttons">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>481</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="start_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(source/icon/Start.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="start_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="stop_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(source/icon/Stop.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="stop_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="clear_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(source/icon/Clear.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="clear_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="exit_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>340</y>
          <width>70</width>
          <height>70</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(source/icon/Exit.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="exit_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>410</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_info">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>701</width>
         <height>361</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTextBrowser" name="info">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>601</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QFrame" name="frame_ip">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>291</width>
          <height>61</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="ip_edit">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>14</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="ip_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>41</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
</string>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_port">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="port_edit">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>14</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="port_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="info_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>211</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baskerville Old Face</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="source/rcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>info</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
